### PR TITLE
Move to precompiled handlebars templates.

### PR DESCRIPTION
**Pull request by matiasdelellis**
In Nextcloud 15 the default Content Security Policy disallows unsafe
eval expressions, so Handlebars templates can no longer be compiled at
runtime.

Fix issue #95

### CODE REVIEW

**Comment by matiasdelellis** [comment]
```
npm install handlebars -g
make js-templates
```

Hi @SlavikCA,
Test in NC15, I think this is enough, :smile: 

Hi @stalker314314 
Check that there is no repression in NC 14. :sweat_smile:

File: lib/Controller/FaceController.php [hunk 1]
```diff
@@ -57,6 +57,7 @@ public function getThumb ($id, $size) {
 		$img = new OCP_Image();
 		$fileName = $ownerView->getLocalFile($path);
 		$img->loadFromFile($fileName);
+		$img->fixOrientation();
```
Comment (stalker314314) [inline lib/Controller/FaceController.php]: sweet:)

**Comment by stalker314314** [comment]
> Check that there is no repression in NC 14.

Sorry, in a week or two, currently OOF. Proceed as if I checked:)

File: lib/Controller/FaceController.php (hunk 1 above)
Comment (matiasdelellis) [reply lib/Controller/FaceController.php]: D'Oh!. 
I mixed that change that i was testing.. 😅
If confirm that fix nc15, I merge it like this.😌

**Pull request merged**